google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -1346,8 +1367,12 @@ PooledSession get(final boolean eligibleForLongRunning) {
           // ignore the exception as it will be handled by the call to super.get() below.
         }
         if (res != null) {
-          res.markBusy(span);
+          res.markBusy(span, openTelemetrySpan);
           span.addAnnotation(sessionAnnotation(res));
+          OpenTelemetryTraceUtil.addEvent(
+              openTelemetrySpan,
+              "Using Session",
+              Attributes.builder().put("sessionId", res.getName()).build());
```

Comment:
Are these the same? (Does `sessionAnnotation(res)` return the same as we are adding to OT?)